app/src/main/java/org/simple/clinic/drugs/selection/custom/drugfrequency/country/EthiopiaDrugFrequencyProvider.kt

```diff
@@ -9,11 +9,11 @@ import org.simple.clinic.drugs.search.DrugFrequency.TDS
 class EthiopiaDrugFrequencyProvider : DrugFrequencyProvider {
   override fun provide(): List<DrugFrequencyChoiceItem> {
     return listOf(
-        DrugFrequencyChoiceItem(drugFrequency = null, labelResId = R.string.custom_drug_entry_sheet_frequency_none),
-        DrugFrequencyChoiceItem(drugFrequency = OD, labelResId = R.string.custom_drug_entry_sheet_frequency_ethiopia_PD),
-        DrugFrequencyChoiceItem(drugFrequency = BD, labelResId = R.string.custom_drug_entry_sheet_frequency_ethiopia_BID),
-        DrugFrequencyChoiceItem(drugFrequency = TDS, labelResId = R.string.custom_drug_entry_sheet_frequency_ethiopia_TID),
-        DrugFrequencyChoiceItem(drugFrequency = QDS, labelResId = R.string.custom_drug_entry_sheet_frequency_ethiopia_QID)
+        DrugFrequencyChoiceItem(drugFrequency = null, label = "None"),
+        DrugFrequencyChoiceItem(drugFrequency = OD, label = "PD"),
+        DrugFrequencyChoiceItem(drugFrequency = BD, label = "BID"),
+        DrugFrequencyChoiceItem(drugFrequency = TDS, label = "TID"),
+        DrugFrequencyChoiceItem(drugFrequency = QDS, label = "QID")
```

Comment:
Why do we need to hard code the values in code? We can inject the `Resources` as a constructor parameter and use that to resolve the resources.